### PR TITLE
Improvements to "There is no spoon - Episode 1" solution based on code review suggestions

### PR DESCRIPTION
**Pull request by en345**
-Fixed the spelling issue
-Excluded unused code

### CODE REVIEW

File: There is no Spoon - Episode 1/en345/There_is_no_spoon_Episode_1.cs [hunk 1]
```diff
@@ -1,8 +1,6 @@
 // # A program that resolves the "There is no spoon - Episode 1" problem from [Coding game website](https://www.codingame.com/home)
```
Comment (bjones1) [inline There is no Spoon - Episode 1/en345/There_is_no_spoon_Episode_1.cs:1]: I would use "solves", not "resolves".

File: There is no Spoon - Episode 1/en345/There_is_no_spoon_Episode_1.cs [hunk 2]
```diff
@@ -14,12 +12,9 @@ class Player
 {
         static void Main(string[] args)
         {
-          
             // ### Get Input Details
-
-            /* <p><i>Get input details includign width, height, and the lines. Use horizontal and vertical lines to store the input details for processing</i></p>
+            /* <p><i>Get input details including width, height, and the lines. Use horizontal and vertical lines to store the input details for processing</i></p>
```
Comment (bjones1) [inline There is no Spoon - Episode 1/en345/There_is_no_spoon_Episode_1.cs:16]: Remove HTML, rewrite more simply using Markdown.

File: There is no Spoon - Episode 1/en345/There_is_no_spoon_Episode_1.cs [hunk 3]
```diff
@@ -1,8 +1,6 @@
 // # A program that resolves the "There is no spoon - Episode 1" problem from [Coding game website](https://www.codingame.com/home)
-
 // ## The problem and the solution
```
Comment (bjones1) [inline There is no Spoon - Episode 1/en345/There_is_no_spoon_Episode_1.cs:2]: Explain what the problem is. There are no details here!

**Review by bjones1 — CHANGES_REQUESTED** [review]

File: There is no Spoon - Episode 1/en345/There_is_no_spoon_Episode_1.cs (hunk 1 above)
Comment (en345) [reply There is no Spoon - Episode 1/en345/There_is_no_spoon_Episode_1.cs]: The two tend to be used interchangeably. I have changed it.

File: There is no Spoon - Episode 1/en345/There_is_no_spoon_Episode_1.cs (hunk 2 above)
Comment (en345) [reply There is no Spoon - Episode 1/en345/There_is_no_spoon_Episode_1.cs]: The issue is that styling was an issue is some cases with markdown. Using HTML seemed to be quicker. I couldn't find a way to resize the image in the solution while improving structuring of the page. Suggestions for resizing the image were to include a CSS file, but that didn't seem to work well with CodeChat. Are CSS files supported in CodeChat?

File: There is no Spoon - Episode 1/en345/There_is_no_spoon_Episode_1.cs (hunk 3 above)
Comment (en345) [reply There is no Spoon - Episode 1/en345/There_is_no_spoon_Episode_1.cs:2]: The details about the problem and the approach used in the solution were part of the image included in the page. Isn't that good enough also?

File: There is no Spoon - Episode 1/en345/There_is_no_spoon_Episode_1.cs (hunk 2 above)
Comment (en345) [reply There is no Spoon - Episode 1/en345/There_is_no_spoon_Episode_1.cs]: I fixed the issue buy resizing the image before adding it to the document.

**Comment by en345** [comment]
@bjones1, I have made changes to the previous version based on your suggestions. However, I also had a question included in one of the replies. 
I had included explanations for the problem and the approach used in solving it in the image included in the document. Will that be good enough? You had commented that there wasn't enough explanations.

**Comment by bjones1** [comment]
Looks good.

**Pull request merged**